canonical-web-and-design/deployment-configs
# Redirect docs.maas.io to maas.io/docs

QA
--

``` bash
openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout self.key -out self.crt -subj "/CN=foo.bar.com"  # Create self-signed certs
microk8s.kubectl create secret tls maas-io-tls --key self.key --cert self.crt  # Make blog.ubuntu.com secret
microk8s.kubectl create secret tls docs-maas-io-tls --key self.key --cert self.crt  # Make blog.ubuntu.com secret
./qa-deploy --production maas.io --tag maas-docs
watch microk8s.kubectl get all,ingress
```

Go to https://docs.maas.io and grab some URLs of some docs to test.

Now edit `/etc/hosts` and add `127.0.0.1 maas.io docs.maas.io`.

Now open a private window in your browser, and go to those docs URLs you saved. Check they redirect to `maas.io/docs`, and you see the right document content.

And check that https://maas.io/docs/2.5 redirects to old-docs.maas.io.
